modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/bigquery/Type.scala

```diff
@@ -12,18 +12,34 @@
  */
 package com.snowplowanalytics.iglu.schemaddl.bigquery
 
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.suggestion.numericType._
 /** BigQuery field type; "array" and "null" are expressed via `Mode` */
 sealed trait Type extends Product with Serializable
 
 object Type {
   case object String extends Type
+
   case object Boolean extends Type
+
   case object Integer extends Type
+
   case object Float extends Type
-  case object Numeric extends Type
+
+  case class Numeric(precision: Int, scale: Int) extends Type
```

Comment:
Are the fields of this case class ever used anywhere?

Reply:
In BQ loader they will be.
```
      case Type.Numeric(precision, scale) if (precision <= 38) & (scale <= 9) => LegacySQLTypeName.NUMERIC
      case _: Type.Numeric => LegacySQLTypeName.BIGNUMERIC
```